lib/src/material/material_controls.dart

```diff
@@ -179,6 +181,21 @@ class _MaterialControlsState extends State<MaterialControls>
       );
     }
 
+    if (chewieController.resolutions != null &&
```

Comment:
Replace with `if (chewieController.resolutions?.isNotEmpty ?? false) {`.

Reply:
thx will do it